### ISSUE TITLE
Running an application that relies on restclient throwing an exception

### ISSUE THREAD

**Issue by Ekultek** [rest-client/rest-client#600]
I'm the author of an application that relies heavily on restclient, as it turns out this application is distributed through a couple OS distos. When running the application with ruby 2.4 it tends to throw this [exception](https://github.com/WhitewidowScanner/whitewidow/issues/55).

```
$ whitewidow
/usr/share/whitewidow/vendor/bundle/ruby/2.4.0/gems/rest-client-2.0.0/lib/restclient/request.rb:198:in `fetch': key not found: :ciphers (KeyError)
        from /usr/share/whitewidow/vendor/bundle/ruby/2.4.0/gems/rest-client-2.0.0/lib/restclient/request.rb:198:in `initialize'
        from /usr/share/whitewidow/vendor/bundle/ruby/2.4.0/gems/rest-client-2.0.0/lib/restclient/request.rb:52:in `new'
        from /usr/share/whitewidow/vendor/bundle/ruby/2.4.0/gems/rest-client-2.0.0/lib/restclient/request.rb:52:in `execute'
        from /usr/share/whitewidow/lib/modules/core/settings.rb:9:in `page'
        from /usr/share/whitewidow/lib/modules/core/settings.rb:18:in `parse'
        from /usr/share/whitewidow/lib/misc/banner.rb:36:in `version_type'
        from /usr/share/whitewidow/lib/imports/constants_and_requires.rb:82:in `<top (required)>'
        from whitewidow.rb:2:in `require_relative'
        from whitewidow.rb:2:in `<main>'
```

Now I've done some research and found that I may need to update the gem, after doing so I had the owner of the distro try to install the package, still getting the same error. Do you have any idea of what's going on?

**Comment by Ekultek** [comment]
Any news on this or updates? It's been almost a week

**Comment by ab** [comment]
This was fixed in rest-client 2.0.1.

https://github.com/rest-client/rest-client/pull/573
